tests/fixtures_server.py

```diff
@@ -170,13 +170,14 @@ async def db_create_tables(override_settings):
     from fractal_server.app.db import DB
     from fractal_server.app.models import SQLModel
 
-    # Calling set_db guarantees that a new pair of sync/async engines is
-    # created every time. This is needed for our Postgresql-based CI, due to
-    # the presence of Enums. See
+    # Calling both set_sync_db and set_async_db every time is needed for our
```

Comment:
Let's keep the information about creating new engines, because this is what really fixed the issue we had in the past.